config/pmix_setup_libev.m4

```diff
@@ -6,39 +6,55 @@
 # Copyright (c) 2017-2019 Research Organization for Information Science
 #                         and Technology (RIST).  All rights reserved.
 # Copyright (c) 2021      Nanook Consulting.  All rights reserved.
+# Copyright (c) 2021      Amazon.com, Inc. or its affiliates.
+#                         All Rights reserved.
 # $COPYRIGHT$
 #
 # Additional copyrights may follow
 #
 # $HEADER$
 #
 
-# MCA_libev_CONFIG([action-if-found], [action-if-not-found])
+# PMIX_LIBEV_CONFIG([action-if-found], [action-if-not-found])
 # --------------------------------------------------------------------
+# Attempt to find a libev package.  If found, evaluate
+# action-if-found.  Otherwise, evaluate action-if-not-found.
+#
+# Modifies the following in the environment:
+#  * pmix_libev_CPPFLAGS
+#  * pmix_libev_LDFLAGS
+#  * pmix_libev_LIBS
+#
+# Adds the following to the wrapper compilers:
+#  * CPPFLAGS: none
+#  * LDLFGAS: add pmix_libev_LDFLAGS
+#  * LIBS: add pmix_libev_LIBS
 AC_DEFUN([PMIX_LIBEV_CONFIG],[
-    PMIX_VAR_SCOPE_PUSH([pmix_libev_dir pmix_libev_libdir pmix_libev_standard_header_location pmix_libev_standard_lib_location pmix_check_libev_save_CPPFLAGS pmix_check_libev_save_LDFLAGS pmix_check_libev_save_LIBS])
+    PMIX_VAR_SCOPE_PUSH([pmix_libev_dir pmix_libev_libdir pmix_check_libev_save_CPPFLAGS pmix_check_libev_save_LDFLAGS pmix_check_libev_save_LIBS])
 
     AC_ARG_WITH([libev],
                 [AS_HELP_STRING([--with-libev=DIR],
                                 [Search for libev headers and libraries in DIR ])])
-    PMIX_CHECK_WITHDIR([libev], [$with_libev], [include/event.h])
-
     AC_ARG_WITH([libev-libdir],
                 [AS_HELP_STRING([--with-libev-libdir=DIR],
                                 [Search for libev libraries in DIR ])])
-    PMIX_CHECK_WITHDIR([libev-libdir], [$with_livev_libdir], [libev.*])
 
-    pmix_libev_support=0
+    pmix_libev_support=1
+
+    AS_IF([test "$with_libev" = "no"],
+          [AC_MSG_NOTICE([Libev support disabled by user.])
+           pmix_libev_support=0])
 
-    AS_IF([test -n "$with_libev" && test "$with_libev" != "no"],
-          [AC_MSG_CHECKING([for libev in])
+    AS_IF([test $pmix_libev_support -eq 1],
+          [PMIX_CHECK_WITHDIR([libev], [$with_libev], [include/event.h])
```

Comment:
I honestly couldn't figure out why you bothered removing it, and it wasn't there before in the libev test, so I didn't add it.  `/a//b` is functionally equivalent to `/a/b`

Reply:
No worries, then - I removed them because that was being done elsewhere in the configure code, so figured it was worth doing for consistency. Probably should remove the code from libevent then so things are consistent - I can do that after this is committed.